src/views/Workflow.vue

```diff
@@ -101,28 +121,39 @@ export default {
      * @type {CylcTree}
      */
     tree: new CylcTree(),
+    table: {},
     isLoading: true,
     // the widgets added to the view
     /**
      * @type {
      *   Object.<string, string>
      * }
      */
-    widgets: {}
+    widgets: {},
+    deltasCallbacks: {}
```

Comment:
Not happy about adding one more property here. I think @oliver-sanders ' initial design had a single subscriptions array/dict. I think that would be better than having `.widgets`, `subscriptions`, `deltasCallbacks`, etc. Will think about it over next days.